src/test/java/core/JenkinsDatabaseSecurityRealmTest.java

```diff
@@ -59,6 +66,54 @@ public void setUp() {
         security.save();
     }
 
+    static void addSessionCookie(Options manager, String path, Date date) {
+        manager.addCookie(new Cookie("JSESSIONID."+Integer.toHexString(new Random().nextInt()),
+                "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+                path,
+                date));
+    }
+
+    @Test
+    public void many_sessions_logout() {
+        User user = realm.signup().fullname(FULL_NAME).email(EMAIL).password(PWD).signup(NAME);
+
+        jenkins.login().doLogin(user.id(), PWD);
+
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.DAY_OF_YEAR, 1);
+        Date tomorrow = calendar.getTime();
+        byte[] array = new byte[7];
+        Options manager = driver.manage();
+        Collections.nCopies(8, 1)
+                .stream()
+                .forEach(i -> addSessionCookie(manager, "/", tomorrow));
+        addSessionCookie(manager, "/will-not-be-sent", tomorrow);
+
+        jenkins.logout();
+
+        StringBuilder builder = new StringBuilder();
+        int sessionCookies = 0;
+
+        final boolean debugCookies = false;
```

Comment:
I think this debug "feature" should be activated by default (and so, removing the variable) as it allow us to better understand a failure in the test if that happens.

Reply:
I wasn't sure how to properly dump output without failing the test.